orchestration/hca_orchestration/solids/load_hca/load_table.py

```diff
@@ -58,6 +58,8 @@ def load_table(
         logging.info(f"No data for metadata type {metadata_type}")
         return None
 
+    remove_empty_blobs(scratch_config.scratch_bucket_name, source_path, gcs_client)
```

Comment:
Dataflow was staging empty blobs for file transfer requests, in addition to those with data. BQ was choking on the empty blobs when importing so we need to remove.